week-1/Homework/extra/1-currency-conversion.js

```diff
@@ -16,20 +16,26 @@ function convertToUSD() {}
   Find a way to add 1% to all currency conversions (think about the DRY principle)
 */
 
-function convertToBRL() {}
+function myMethod(number) {
+  var myNumber = number.toFixed(2);
+  var sign = "£";
+  var result = sign + myNumber;
+  return result;
+}
```

Comment:
You can not change the name of the function because it has been used in the test and you forget to complete the other function.